zk-sdk/src/encryption/elgamal.rs

```diff
@@ -6,9 +6,9 @@
 //! A twisted ElGamal ciphertext consists of two components:
 //! - A Pedersen commitment that encodes a message to be encrypted
 //! - A "decryption handle" that binds the Pedersen opening to a specific public key
-//! In contrast to the traditional ElGamal encryption scheme, the twisted ElGamal encodes messages
-//! directly as a Pedersen commitment. Therefore, proof systems that are designed specifically for
-//! Pedersen commitments can be used on the twisted ElGamal ciphertexts.
+//!   In contrast to the traditional ElGamal encryption scheme, the twisted ElGamal encodes messages
```

Comment:
Nit, the first line above the diff should probably have a period at the end
`to a specific public key` TO `to a specific public key.`